src/connected/Workspace.tsx

```diff
@@ -117,7 +117,32 @@ const Workspace: React.FC<WorkspaceProps> = ({
                                              }) => {
     const [map, setMap] = React.useState<OlMap | null>(null);
     const [layout, setLayout] = React.useState<Layout>(getLayout());
+    const useBeforeRender = (callback: any) => {
+        const [isRun, setIsRun] = React.useState(false);
 
+        if (!isRun) {
+            callback();
+            setIsRun(true);
+        }
+    };
+
+    useBeforeRender(() => {
+        window.addEventListener("error", (e) => {
+            if (e) {
```

Comment:
If I'm understanding this code correctly, it's meant to work around the error by catching it and hiding the elements that display the error message. No problem if that's the most practical solution, but I'd like at least a short comment mentioning it. If it's caused by a reported bug in an external library, a link to the issue there would also be helpful.

One thing that concerns me is that `if (e)` presumably catches *any* error, so I'm worried that it might silently swallow errors unrelated to this bug. Can make the condition more specific, to only match the specific error message we're expecting here?

Reply:
Yes that's the possible workaround. Its not caused because of any external library but it causes when we resize the window to arrange map and timeseries. First when we load the page its only map and then we plot points to add timeseries the window is resized. So at that time we get this resize error because what happens is the complete screen is loaded but all the controls are not resized properly. In a case this error is not harmful or it will not break our running page. So  I found this solution/workaround quite reasonable after reading through multiple stuffs was this. 
I referred to the below link for this workaround. Many of them faces issue due to specific library but its not the case for us. I debugged our code and the we were getting this issue when window.resize was called.

https://stackoverflow.com/questions/75774800/how-to-stop-resizeobserver-loop-limit-exceeded-error-from-appearing-in-react-a

For if(e), yes we can be specific. Though I think the later conditions would not be satisfied but to be on safer side I can add precise error condition.

Reply:
Yes, please add a link to the SO question if it informed your implementation, and make the `if` condition more specific. (Apart from being safer, checking the error name also implicitly documents the problem that we're trying to fix here :) ). After that I'll be happy to approve.

Reply:
Yes done, link I have already mentioned  in my answer and yes did changed the condition for if(e) :)